source/concepts/xml-files.rst

```diff
@@ -0,0 +1,57 @@
+XML Files
```

Comment:
I think the title should reflect the idea that this is the XML format for OpenMM simulations specifically.

Reply:
Done in this commit [53f559a](https://github.com/IRL2/nanover-docs/pull/115/commits/53f559a4555a4359a66443667809392abf6f96e3).